src/main.cpp

```diff
@@ -705,6 +741,20 @@ int main(int argc, char *argv[])
                 }
                 break;
 
+	    case 'P':
+                param = optarg;
```

Comment:
This code is shared with the video parameter parsing, can we have a separate function like:

```
static void parse_codec_option(std::map<std::string, std::string>& options, std::string param)
{
...
}
```